src/app/components/BCP47Tag.svelte

```diff
@@ -2,25 +2,36 @@
   import InputField from "./InputField.svelte";
   import * as bcp47 from "bcp-47";
   import worker from "../spawn-worker";
+  import { createEventDispatcher } from "svelte";
 
   let errorMessage: string = "";
+  const dispatch = createEventDispatcher();
 
-  function validateAndSetLanguage(event: CustomEvent) {
+  function onInputValue(event: CustomEvent) {
```

Comment:
Why change the name to `onInputValue()` when `validateAndSetLanguage()` is more self-documenting?

Reply:
I believe this change came from @IanSeng  PR #165,